src/internal/account.ts

```diff
@@ -795,3 +803,104 @@ export async function isAccountExist(args: { aptosConfig: AptosConfig; authKey:
     throw new Error(`Error while looking for an account info ${accountAddress.toString()}`);
   }
 }
+
+/**
+ * Verifies a Secp256k1 account by checking the signature against the message and account's authentication key.
+ *
+ * This function takes a message and signature, and attempts to recover the public key that created the signature.
+ * It then verifies that the recovered public key matches the authentication key stored on-chain for the provided account address.
+ *
+ * If a recovery bit is provided, it will only attempt verification with that specific recovery bit.
+ * Otherwise, it will try all possible recovery bits (0-3) until it finds a match.
+ *
+ * @param args - The arguments for verifying the Secp256k1 account
+ * @param args.aptosConfig - The configuration for connecting to the Aptos blockchain.
+ * @param args.message - The message that was signed
+ * @param args.signature - The signature to verify (either raw hex or Secp256k1Signature object)
+ * @param args.recoveryBit - Optional specific recovery bit to use for verification
```

Comment:
Can we ever expect the recoveryBit to be provided manually by the user?
How would I obtain it when signing a message using a Secp256k1PrivateKey

Reply:
It's unclear to me.  We dont provide in our APIs but other SDK may.  It is optional since there are a finite number of bit options and you can simply loop if not provided.